cl_sii/cte/f29/parse_datos_obj.py

```diff
@@ -22,27 +22,50 @@
 CTE_F29_DATOS_OBJ_SCHEMA = read_json_schema(_CTE_F29_DATOS_OBJ_SCHEMA_PATH)
 
 
-def parse_sii_cte_f29_datos_obj(datos_obj: SiiCteF29DatosObjType) -> CteForm29:
+def parse_sii_cte_f29_datos_obj(
+    datos_obj: SiiCteF29DatosObjType,
+    validate_schema: Optional[Callable[[SiiCteF29DatosObjType], None]] = None,
```

Comment:
Perhaps `schema_validator` would be more appropriate?

Reply:
At first I used `schema_validator`, but given that it's a function, I thought that `validate_schema(datos_obj)` looked better than `schema_validator(datos_obj)`.

What do you think?

Reply:
Any conventions that we can refer to? I think in Django the arguments for callables such as validators are named "validators"

Reply:
> Any conventions that we can refer to?

The validators in `cl_sii.dte.data_models` are named `validate_*`.

https://github.com/fyntex/lib-cl-sii-python/blob/228a626381363f097b3e282a9b9c8ded438165a3/cl_sii/dte/data_models.py#L33-L97

Reply:
IMO you are confusing the arg/param name with the function name